gradle/libs.versions.toml

```diff
@@ -108,4 +106,4 @@ hilt-plugin = { id = "com.google.dagger.hilt.android", version.ref = "hilt" }
 baselineprofile = { id = "androidx.baselineprofile", version.ref = "baselineprofile" }
 
 [bundles]
-retrofitBundle = ["retrofit", "retrofit-kotlin-serialization", "okhttp-interceptor", "okhttp-mockserver"]
+retrofit = ["retrofit", "retrofit-kotlinSerialization", "okhttp-loggingInterceptor"]
```

Comment:
retofitBundle makes it easier to understand at a glance which dependencies are grouped together.

Reply:
and correct as per naming convention

Reply:
I thought it was redundant because it's already part of the bundle namespace:
`bundle.retrofitBundle`

Reply:
for code readability is easy to know that it bundled